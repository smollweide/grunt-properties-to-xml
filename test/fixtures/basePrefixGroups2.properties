# base i18n properties
group2.item2 = Item 2
group1.item1 = Item 1
group1.item2 = Item 2
group2.item3 = Item 3 = something
group1.item3 = Item 3 = something
group2.item1 = Item 1