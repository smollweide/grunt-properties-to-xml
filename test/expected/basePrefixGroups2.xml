<group name="group2">
	<label key="react.item2">Item 2</label>
	<label key="react.item3">Item 3 = something</label>
	<label key="react.item1">Item 1</label>
</group>
<group name="group1">
	<label key="react.item1">Item 1</label>
	<label key="react.item2">Item 2</label>
	<label key="react.item3">Item 3 = something</label>
</group>